openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -340,3 +341,10 @@ def pre_requisite_courses(self):
         Returns a list of ID strings for this course's prerequisite courses.
         """
         return json.loads(self._pre_requisite_courses_json)
+
+    @classmethod
+    def get_all_courses(cls):
+        """
+        Returns all courses from course overviews.
+        """
+        return [course_overview.id for course_overview in cls.objects.all()]
```

Comment:
That makes sense.  All we need is the course IDs.  Do you know what the preferred nomenclature is?  I often see course keys and IDs used interchangeably.